crypto/evp/evp_fetch.c

```diff
@@ -504,6 +504,11 @@ char *evp_get_global_properties_str(OSSL_LIB_CTX *libctx, int loadconfig)
     char *propstr = NULL;
     size_t sz;
 
+    if (plp == NULL) {
```

Comment:
Can we ossl_assert?

Reply:
I think it should return an empty string in this case.